src/commons/components/create-pool/create-pool.js

```diff
@@ -87,22 +87,31 @@
         function init() {
 
             if (typeof $rootScope.clusterData !== "undefined") {
-                vm.clusterList = $rootScope.clusterData.clusters;
+                getCephClusterList();
             } else {
                 utils.getObjectList("Cluster")
                 .then(function(data) {
                     $rootScope.clusterData = data;
-                    vm.clusterList = $rootScope.clusterData.clusters;
+                    getCephClusterList();
                 });
             }
 
-            vm.selectedCluster = vm.clusterList[0];
+            vm.selectedCluster = vm.cephClusterList[0];
             vm.selectedType = vm.poolType[0];
             vm.selectedEC = vm.ECType[0];
             vm.selectedOwner = vm.owners[0];
             vm.updatePoolName();
         }
 
+        function getCephClusterList(){
+            var index;
+            for(index = 0 ; index < $rootScope.clusterData.clusters.length ; index++) {
```

Comment:
@cloudbehl Please store the length in some variable and then use it in for loop. This impacts the performance.